example/service/dynamodb/scanItems/README.md

```diff
@@ -44,3 +45,9 @@ In DynamoDB the structure of the item to be returned will be:
 	- "Value 1": abc
 	- "Value 2": 1234567890
 ```
+
+## Credentials
+The easiest way to set the credentials and start using this example is to install the AWS cli application: http://docs.aws.amazon.com/cli/latest/userguide/installing.html
```

Comment:
I am also wondering if this is the right place to put this. Perhaps, take this out and maybe adding it to our dev guide may be a better option.